docs/source/resources-algorithms.rst

```diff
@@ -3,4 +3,386 @@ OpenDCS Algorithms - Resources
 ###################################
 
 
-Content coming soon ...
+****************************
+Algorithms Resources - Recap
+****************************
+
+There are a number of "Standard" or "Baseline" algorithms that come 
+with OpenDCS when it is installed.  These can be thought of as banked 
+algorithms or functions that can be called upon when setting up a computation.
+These standard algorithms are generally basic arithmetic or time series
+manipulation or transformations.  For example, the average algorithm
+can be used to create a daily or monthly time series from an hourly or 
+daily time series, respectively.  In addition to simple arithmetic,
+or transformations, there are some set defined hydrologic specific
+algorithms that are commonly used in water management. For example
+there is an algorithm that can be called upon to calculate an incremental
```

Comment:
suggestion:  give name of algorithm , instead of 'there is an algorithm', use 'the X algorithm.

Reply:
just revised with a couple examples